Dockerfile.plugins.demo2

```diff
@@ -0,0 +1,22 @@
+# This image is used for testing OpenShift Console dynamic plugin capabilities.
+#
+# See frontend/dynamic-demo-plugin/README.md for details.
+
+# Stage 0: build the demo plugin
```

Comment:
Let's update the documentation for this dockerfile to let users know that the the plugin must be pre-built:
```suggestion
# Stage 0: build the demo plugin
# Manually pre-build the demo plugin:
# cd frontend
# yarn install
# cd dynamic-demo-plugin
# yarn install
# yarn build
```